Nos/Views/Profile/Edit/ProfileEditView.swift

```diff
@@ -166,15 +166,15 @@ struct ProfileEditView: View {
 
     private func populateTextFields() {
         viewContext.refresh(author, mergeChanges: true)
-        nameText = author.name ?? author.displayName ?? ""
+        displayNameText = author.displayName ?? author.name ?? ""
```

Comment:
Hey @pelumy, this looks good and accomplishes what I asked for in my last comment. However I found one weird flow on my test account that I would like to smooth out before we merge. It looks like if the user has only set a `name` in Nos but never a `displayName` then we actually show an empty `displayName` field. I think this is happening because we have been saving an empty string into the `displayName` field, so this line of code ends up evaulating as `displayNameText = ""`. It looks a little weird:


https://github.com/user-attachments/assets/bdbce428-87d0-40ea-a1ef-52136bc9aed1


Can you tweak this so that it uses the name if `author.displayName` is nil _or empty_?

Reply:
Ooh, thanks for catching this @mplorentz . I will push a fix for this.